docs/UserGuide.md

```diff
@@ -198,17 +201,21 @@ Format: `list`
 
 ### Reviewing all flashcards: `review`
 
-Reviews the current list of flashcards. This puts the user in review mode and the user can no longer input commands to the textbox.
+Review the current list of flashcards. This puts the user in review mode and the user can no 
```

Comment:
Maybe you can consider using "Reviews" instead of "Review" so that our UG has a standardised format.